src/Microsoft.DotNet.Build.Tasks/PackageFiles/packageresolve.targets

```diff
@@ -183,4 +183,7 @@
     <Message Importance="Low"
              Text="Removed all ResolvedTagetingPackReferences that were not specified explicitly as a TargetingPackReference=[@(TargetingPackReference)]. PackageReferencesToRemove=[@(PackageReferencesToRemove)]." />
   </Target>
+  
+  <Import Project="$(CustomAfterPackageResolveTargets)" Condition="'$(CustomAfterPackageResolveTargets)' != '' AND Exists('$(CustomAfterPackageResolveTargets)')"/>
```

Comment:
Lets make this a little more general and put it in build.common.targets.

Also the Exists check should be enough as you don't need to check for empty as well.